### PR TITLE
PNC Neonatal Danger Sign Implementation

### LINKED ISSUE #12: Implement Child Danger Signs
Image

[Child Danger Signs](https://docs.google.com/spreadsheets/d/1dr8uhaF9KN09DzcQf6-l_VsC6jXCQ5ddj1cKnuymkto/edit#gid=179989320&range=A4)

### PR DESCRIPTION
**Pull request by devcolonel**
Close #12

### CODE REVIEW

File: opensrp-chw/src/nacp/assets/json.form-sw/pnc_danger_signs_baby.json [hunk 1]
```diff
@@ -167,18 +176,168 @@
...
               }
             ]
           }
         }
+      },
+      {
+        "key": "neonatal_visited_health_facility",
+        "type": "native_radio",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "114403AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
```
Comment (cozej4) [inline opensrp-chw/src/nacp/assets/json.form-sw/pnc_danger_signs_baby.json]: We are phasing out the use of openmrs uuids here, please add the same value as the key

File: opensrp-chw/src/nacp/assets/json.form-sw/pnc_danger_signs_baby.json [hunk 2]
```diff
@@ -167,18 +176,168 @@
...
+        "openmrs_entity_parent": "",
+        "label": "Je, katika siku 3 zilizopita, umempeleka mtoto kwenye kituo cha afya kwa mojawapo ya hali hizi zilizotambuliwa?",
+        "options": [
+          {
+            "key": "yes",
+            "openmrs_entity": "",
+            "openmrs_entity_id": "1065AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
```
Comment (cozej4) [inline opensrp-chw/src/nacp/assets/json.form-sw/pnc_danger_signs_baby.json]: We are phasing out the use of openmrs uuids here, please add the same value as the key

File: opensrp-chw/src/nacp/assets/json.form-sw/pnc_danger_signs_baby.json [hunk 3]
```diff
@@ -167,18 +176,168 @@
...
+            "openmrs_entity_parent": "",
+            "text": "Ndio"
+          },
+          {
+            "key": "no",
+            "openmrs_entity": "",
+            "openmrs_entity_id": "1066AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
```
Comment (cozej4) [inline opensrp-chw/src/nacp/assets/json.form-sw/pnc_danger_signs_baby.json]: We are phasing out the use of openmrs uuids here, please add the same value as the key

File: opensrp-chw/src/nacp/assets/json.form-sw/pnc_danger_signs_baby.json [hunk 4]
```diff
@@ -167,18 +176,168 @@
...
+              "rules-file": "pnc_danger_signs_baby_relevance.yml"
+            }
+          }
+        }
+      },
+      {
+        "key": "neonatal_referral_health_facility",
+        "type": "native_radio",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "114403AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
```
Comment (cozej4) [inline opensrp-chw/src/nacp/assets/json.form-sw/pnc_danger_signs_baby.json]: We are phasing out the use of openmrs uuids here, please add the same value as the key

File: opensrp-chw/src/nacp/assets/json.form-sw/pnc_danger_signs_baby.json [hunk 5]
```diff
@@ -167,18 +176,168 @@
...
+        "openmrs_entity_parent": "",
+        "label": "Kulingana na mazungumzo uliyofanya na Mama, unafikiri Mtoto anahitaji kurudi kwenye kituo kwa uchunguzi zaidi?",
+        "options": [
+          {
+            "key": "yes",
+            "openmrs_entity": "",
+            "openmrs_entity_id": "1065AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
```
Comment (cozej4) [inline opensrp-chw/src/nacp/assets/json.form-sw/pnc_danger_signs_baby.json]: We are phasing out the use of openmrs uuids here, please add the same value as the key

File: opensrp-chw/src/nacp/assets/json.form-sw/pnc_danger_signs_baby.json [hunk 6]
```diff
@@ -167,18 +176,168 @@
...
+            "openmrs_entity_parent": "",
+            "text": "Hapana"
+          },
+          {
+            "key": "no",
+            "openmrs_entity": "",
+            "openmrs_entity_id": "1066AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
```
Comment (cozej4) [inline opensrp-chw/src/nacp/assets/json.form-sw/pnc_danger_signs_baby.json]: We are phasing out the use of openmrs uuids here, please add the same value as the key

File: opensrp-chw/src/nacp/assets/json.form/pnc_danger_signs_baby.json [hunk 7]
```diff
@@ -148,14 +157,60 @@
...
+              }
+            ]
+          }
+        }
+      },
+      {
+        "key": "neonatal_visited_health_facility",
+        "type": "native_radio",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "114403AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
```
Comment (cozej4) [inline opensrp-chw/src/nacp/assets/json.form/pnc_danger_signs_baby.json]: We are phasing out the use of openmrs uuids here, please add the same value as the key

File: opensrp-chw/src/nacp/assets/json.form/pnc_danger_signs_baby.json [hunk 8]
```diff
@@ -148,14 +157,60 @@
...
+        "openmrs_entity_parent": "",
+        "label": "Have you, in the past 3 days, taken the baby to a health facility for any of these identified conditions?",
+        "options": [
+          {
+            "key": "yes",
+            "openmrs_entity": "",
+            "openmrs_entity_id": "1065AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
```
Comment (cozej4) [inline opensrp-chw/src/nacp/assets/json.form/pnc_danger_signs_baby.json]: We are phasing out the use of openmrs uuids here, please add the same value as the key

File: opensrp-chw/src/nacp/assets/json.form/pnc_danger_signs_baby.json [hunk 9]
```diff
@@ -148,14 +157,60 @@
...
+            "openmrs_entity_parent": "",
+            "text": "Yes"
+          },
+          {
+            "key": "no",
+            "openmrs_entity": "",
+            "openmrs_entity_id": "1066AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
```
Comment (cozej4) [inline opensrp-chw/src/nacp/assets/json.form/pnc_danger_signs_baby.json]: We are phasing out the use of openmrs uuids here, please add the same value as the key

File: opensrp-chw/src/nacp/assets/json.form/pnc_danger_signs_baby.json [hunk 10]
```diff
@@ -167,17 +222,121 @@
...
+        "type": "native_radio",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "114403AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+        "openmrs_entity_parent": "",
+        "label": "Based on the discussion do you think the Baby needs to return to the facility for a further checkup?",
+        "options": [
+          {
+            "key": "yes",
+            "openmrs_entity": "",
+            "openmrs_entity_id": "1065AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
```
Comment (cozej4) [inline opensrp-chw/src/nacp/assets/json.form/pnc_danger_signs_baby.json]: We are phasing out the use of openmrs uuids here, please add the same value as the key

File: opensrp-chw/src/nacp/assets/json.form/pnc_danger_signs_baby.json [hunk 11]
```diff
@@ -167,17 +222,121 @@
...
+              "rules-file": "pnc_danger_signs_baby_relevance.yml"
+            }
+          }
+        }
+      },
+      {
+        "key": "neonatal_referral_health_facility",
+        "type": "native_radio",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "114403AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
```
Comment (cozej4) [inline opensrp-chw/src/nacp/assets/json.form/pnc_danger_signs_baby.json]: We are phasing out the use of openmrs uuids here, please add the same value as the key

File: opensrp-chw/src/nacp/assets/json.form/pnc_danger_signs_baby.json [hunk 12]
```diff
@@ -167,17 +222,121 @@
...
+            "key": "yes",
+            "openmrs_entity": "",
+            "openmrs_entity_id": "1065AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+            "openmrs_entity_parent": "",
+            "text": "Yes"
+          },
+          {
+            "key": "no",
+            "openmrs_entity": "",
+            "openmrs_entity_id": "1066AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
```
Comment (cozej4) [inline opensrp-chw/src/nacp/assets/json.form/pnc_danger_signs_baby.json]: We are phasing out the use of openmrs uuids here, please add the same value as the key

**Review by cozej4 — CHANGES_REQUESTED** [review]

**Review by gosso22 — APPROVED** [review]

**Comment by gosso22** [comment]
@cozej4 - this one is ready for review

**Review by cozej4 — APPROVED** [review]

**Pull request merged**